virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -68,6 +68,14 @@ This release adds new features and enhancements related to the following compone
 [id="virt-4-17-virtualization"]
 === Virtualization
 
+//CNV-34780 GA: Safe memory overcommitment using WASP
+* You can now increase VM workload density on nodes by overcommitting memory (RAM) with the `wasp-agent`. The wasp agent assigns swap resources to worker nodes and manages pod evictions when nodes are at risk.
```

Comment:
Looks like the first instance of "virtual machine (VM)" is in the bullet after this one (line 80). So maybe here:
```suggestion
* You can now increase virtual machine (VM) workload density on nodes by overcommitting memory (RAM) with the `wasp-agent`. The wasp agent assigns swap resources to worker nodes and manages pod evictions when nodes are at risk.
```
And change the instance on line 80 to just "VM"?

Reply:
@adellape Ah, actually we have "virtual machines (VMs)" on line 17, and it's written as VM on line 66. I think we struggle with how often to write it out. I might leave it as-is in this PR.